admin/app/components/solidus_admin/orders/index/component.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Orders::Index::Component < SolidusAdmin::BaseComponent
+  def initialize(
+    page:,
+    badge_component: component('ui/badge'),
+    table_component: component('ui/table'),
+    pagination_component: component('ui/table/pagination'),
+    button_component: component("ui/button")
+  )
+    @page = page
+
```

Comment:
Super-nit: we can remove this blank line :slightly_smiling_face:

Reply:
The blank line was introduced to help visually separate the initialization of injectable components from other elements 😄
However, I noticed that was already removed in `products/index`.
In the interest of consistency, I'll remove it here too 👍